ROME_NAF/analyses/print_array.py

```diff
@@ -0,0 +1,140 @@
+import pandas as pd
+
+# Affichage des codes ROME les plus fréquents
+
+ROME_df = pd.read_csv('../referentiels/referentiel_ROME/20150921_arboprincipale28427_ROME.csv', index_col=0, sep='|', dtype=str)
+OGR_df = pd.read_csv('../referentiels/referentiel_OGR/20150921_arboprincipale28427_OGR.csv', sep='|', dtype=str).set_index('OGR')
+NAF_df = pd.read_csv('../referentiels/referentiel_NAF/naf2008_liste_n5_nouveau_header.csv', sep='|', encoding="utf-8").set_index(['NAF'])
+
+stats_lbb = pd.read_csv('../stats_lbb/stats_lbb.csv')
+top_slugs = stats_lbb[:34]
+
+def pprint_ROME(array):
+    i = 0
+    for l in top_slugs.iterrows():
```

Comment:
Pour info tu peux faire `for i, l in enumerate(top_slugs.iterrows()):` ça évite de déclarer `i ` juste au dessus et de l'incrémenter à la main juste en dessous.

Reply:
merci !